source/clustering.md

```diff
@@ -174,7 +177,7 @@ from myst_nb import glue
 import pandas as pd
 
 data = pd.read_csv(
-    'data/toy_penguins.csv'
+    'data/penguins_toy.csv'
```

Comment:
i'm trying to change the naming convention throughout to `[dataset-slug]_style.csv` to make the datasets sort nicely by filename. i'm just doing this as i go through, not a full pass on the whole repo

Reply:
(similar change happened in R)